tests/integration/ledger_sync/ls_outboundq_tests.py

```diff
@@ -78,23 +70,85 @@ def test_role_outq_insertion():
         role_response = create_test_role(session, role_payload)
         assert_api_success(role_response)
 
-        inserted_queue_item = peek_at_q_unfiltered("outbound_queue")
-        LOGGER.info(
-            "Received queue entry %s from outbound queue...", inserted_queue_item["id"]
-        )
-        successful_insert = bool(inserted_queue_item)
-        assert expected_result == successful_insert
+        outbound_queue_data = {
+            "description": "Test Unique Role 1",
+            "name": "TestUniqueRole0501201903",
+            "group_types": -2147483646,
+            "members": [],
+            "owners": "",
+            "remote_id": "",
+        }
+        expected_payload = {
+            "data": outbound_queue_data,
+            "data_type": "group",
+            "provider_id": "NEXT-created",
+            "status": "UNCONFIRMED",
+            "action": "",
+        }
 
-        # Check status of new outbound_entry
-        role_entry = get_role("TestUniqueRole0501201903")
-        outbound_queue_data = prepare_outbound_queue_data(role_entry[0], "role")
         outbound_entry = get_outbound_queue_entry(outbound_queue_data)
-        assert outbound_entry[0]["status"] == "UNCONFIRMED"
+        outbound_entry[0].pop("timestamp")
+        outbound_entry[0].pop("id")
+        assert outbound_entry[0] == expected_payload
 
         delete_role_by_name("TestUniqueRole0501201903")
         delete_user_by_username("testuniqueuser0501201901")
 
 
+@pytest.mark.skipif(
+    ENABLE_LDAP_SYNC == 0, reason="Skipping test, LDAP mode is not enabled"
```

Comment:
Not a blocker, just a note for the near future: we may want to add an LDAP container to our test docker-compose file and remove the need for flags like this in our tests.